serde/Cargo.toml

```diff
@@ -16,6 +15,7 @@ rust-version = "1.31"
 
 [dependencies]
 serde_derive = { version = "1", optional = true, path = "../serde_derive" }
+serde_core = {version = "1", path = "../serde_core", default-features = false }
```

Comment:
This needs a "=1.0.Z" constraint because otherwise the serde_derive constraint below on line 40 is ineffective. A version of serde_derive could get mixed with a nonmatching version of serde_core that contains the wrong helper types.